### DISCUSSION TITLE
(virtualbox) Split out the VirtualBox Extension into it's own package

### DISCUSSION THREAD

**Discussion by pauby** [chocolatey-community/chocolatey-packages#2136] [Ideas]
The VirtualBox package installs the VirtualBox extensions from inside the package. If the extension package is updated outside the release of VirtualBox, we are forced to update the VirtualBox package. This feels wrong. This [PR](https://github.com/chocolatey-community/chocolatey-packages/pull/2135) and this [issue](https://github.com/chocolatey-community/chocolatey-packages/issues/2132) have been raised about it recently.

We should consider splitting the VirtualBox extensions out to a separate package.

@AdmiringWorm we had a quick conversation about this, but wonder if you could add your comments here. If there are no objections, I'll raise an issue to pick this up in the next week or so.

**Comment by TheCakeIsNaOH** [comment]
Duplicate of discussion https://github.com/chocolatey-community/chocolatey-packages/discussions/1961

**Comment by AdmiringWorm** [comment]
As TheCakeIsNaOH already mentioned, this is a duplicate. Let us keep the discussion in a single discussion thread.
 
I'll add my thoughts to that thread and close this as a duplicate.
